lib/common.sh

```diff
@@ -78,12 +106,21 @@ eecho()
 vecho()
 {
     color=$NORMAL
+    version=$1
+
+    if [[ $version == $LOG_V3 || $version == $LOG_V4 ]]; then
+        prefix="NFS${version}"
+        msg=$2
+    else
+        prefix=""
+        msg=$1
+    fi
 
     # Unless AZNFS_VERBOSE flag is set, do not echo to console.
     if [ -z "$AZNFS_VERBOSE" -o "$AZNFS_VERBOSE" == "0" ]; then
         (
             flock -e 999
-            echo -e "$(date -u +"%a %b %d %G %T.%3N") $HOSTNAME $$: ${color}${*}${NORMAL}" >> $LOGFILE
+            echo -e "${prefix}: $(date -u +"%a %b %d %G %T.%3N") $HOSTNAME $$: ${color}${msg}${NORMAL}" >> $LOGFILE
```

Comment:
pls note that currently all the log functions support multiple arguments, so both the following will have the same effect

vecho this is the logline
vecho "this is the logline"

this means, you will have to use ${*}